extensions/resteasy-classic/rest-client-config/runtime/src/main/java/io/quarkus/restclient/config/RestClientsConfig.java

```diff
@@ -89,6 +90,16 @@ public interface RestClientsConfig {
      */
     Optional<String> nonProxyHosts();
 
+    /**
+     * Proxy connection timeout
+     * <p>
+     * Can be overwritten by client-specific settings.
+     * <p>
+     * This property is not applicable to the RESTEasy Client.
+     */
+    @ConfigDocDefault("10s")
```

Comment:
Why not use `@WithDefault` instead?

Reply:
Because I want to rely on Vertx default handling